src/Umbraco.Core/Logging/Viewer/JsonLogViewer.cs

```diff
@@ -105,5 +114,21 @@ protected override IReadOnlyList<LogEvent> GetLogs(DateTimeOffset startDate, Dat
             return logs;
         }
 
+        private bool TryRead(LogEventReader reader, out LogEvent evt)
+        {
+            try
+            {
+                return reader.TryRead(out evt);
+            }
+            catch (JsonReaderException ex)
+            {
+                // As we are reading/streaming one line at a time in the JSON file
+                // Thus we can not rpeort the line number, as it will always be 1
+                _logger.Error<JsonLogViewer>(ex, "Unable to parse a line in the JSON log file");
```

Comment:
Do we have any method to get more information. E.g line number or something like that

Reply:
Well maybe my comment not 100% clear - but as JSONReader from the 3rd Party Serilog Nuget package reads the stream,  so its read one line in and then tries to parse it as JSON. So the line number will in the exception will always be line one